easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0-libxml2-2.9.7.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.8.1'
+libxml2_ver = '2.9.7'
+versionsuffix = '-libxml2-%s' % libxml2_ver
+
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
+build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
+and documentation"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ff942af0e438ced4a8b0ea4b0b6e0d6d657157c5e2364de57baa279c1c125c43']
+
+dependencies = [
+    ('libxml2', libxml2_ver),
+    ('ncurses', '6.0'),
+]
+
+# use same binutils version that was used when building GCCcore toolchain
+builddependencies = [('binutils', '2.28')]
+
+configopts = '--without-emacs --with-libxml2-prefix=$EBROOTLIBXML2'
+
+moduleclass = 'vis'
```

Comment:
@boegel very minor nitpick - why is the `moduleclass` `vis`?

Reply:
That's a very good question, it's clearly wrong...

I suggest we fix this in a followup PR (and also add a custom `sanity_check_paths` to the `gettext` easyconfigs, while we're at it)?

Reply:
Yes, agreed. I'll merge this one for now.